src/test/java/com/redhat/jenkins/plugins/integration/CachetGatingPluginTest.java

```diff
@@ -284,7 +284,32 @@ public void triggerBuildGatingStep() throws Exception {
         for (String s: log) {
             System.out.println(s);
         }
-        j.assertLogContains("Major Outage", b2);
+        j.assertLogContains(ResourceStatus.MAJOR_OUTAGE.toString(), b2);
+    }
+
+    @Test
+    public void triggerBuildGatingOperationalStep() throws Exception {
+        WorkflowJob p = j.createProject(WorkflowJob.class, "triggerBuildGatingOperationalStep");
+        p.addProperty(new CachetJobProperty(true, Arrays.asList("errata")));
+        p.setDefinition(new CpsFlowDefinition(loadPipelineScript("simple-with-step.groovy"), true));
+        stubFor(normal);
+        ResourceUpdater.setResources();
+
+        QueueTaskFuture<WorkflowRun> item = p.scheduleBuild2(0);
+        assertNotNull(item);
+
+        WorkflowRun b2 = item.get();
+        j.assertBuildStatusSuccess(j.waitForCompletion(b2));
+        List<String> log = b2.getLog(1000);
+        for (String s: log) {
```

Comment:
nit: Not nice to do in test, `assertLogContains` is going to print this in case of failure so probably can be avoided.